did/base.py

```diff
@@ -125,6 +124,17 @@ def quarter(self):
                 f"Invalid quarter start '{month}', should be integer.")
         return month
 
+    @property
+    def week(self):
```

Comment:
Why is this called `week` when it is really the first day of the week?

```suggestion
    def first_day_of_week(self) -> int:
```